model/task_config.go

```diff
@@ -119,7 +119,7 @@ func (c *TaskConfig) GetWorkingDirectory(dir string) (string, error) {
 
 // S3Path returns the path to the working directory dump in S3 for a task.
 func (c *TaskConfig) S3Path() string {
-	return filepath.Join(c.ProjectRef.Identifier, c.Task.Version, c.Task.BuildVariant, c.Task.DisplayName, "latest")
+	return filepath.Join(c.Task.Project, c.Task.Version, c.Task.BuildVariant, c.Task.DisplayName, "latest")
```

Comment:
shouldn't this be the same?

Reply:
It won't be on Windows. I don't want an agent on Windows using `\` for the paths when the path has to be os-independent.